packages/valory/services/trader/service.yaml

```diff
@@ -247,7 +247,7 @@ type: skill
         tool_punishment_multiplier: ${TOOL_PUNISHMENT_MULTIPLIER:int:1}
         redeem_round_timeout: ${REDEEM_ROUND_TIMEOUT:float:3600.0}
         contract_timeout: ${CONTRACT_TIMEOUT:float:300.0}
-        file_hash_to_strategies_json: ${FILE_HASH_TO_STRATEGIES_JSON:list:[["bafybeihufqu2ra7vud4h6g2nwahx7mvdido7ff6prwnib2tdlc4np7dw24",["bet_amount_per_threshold"]],["bafybeif55cu7cf6znyma7kxus4wxa2doarhau2xmndo57iegshxorivwmq",["kelly_criterion"]]]}
+        file_hash_to_strategies_json: ${FILE_HASH_TO_STRATEGIES_JSON:list:[["bafybeihufqu2ra7vud4h6g2nwahx7mvdido7ff6prwnib2tdlc4np7dw24",["bet_amount_per_threshold"]],["bafybeif55cu7cf6znyma7kxus4wxa2doarhau2xmndo57iegshxorivwmq",["kelly_criterion"]],["bafybeihcm4mmf2qqhe3ly47rxtmhcjxrlnnujwyzkcsq6jkbnu6ojxrwja",["new_kelly_criterion"]]]}
```

Comment:
Remove the old kelly here. Moreover, you are still using the `kelly_criterion` as a trading strategy above.

Reply:
The hash is incorrect, it points to the executable instead of the custom package.

Reply:
where is the hash for the custom package?

Reply:
It is in the output of the command that publishes it to IPFS:

`autonomy ipfs add`

Reply:
ok found it. Creating a new commit with changes